include/FreeRTOS.h

```diff
@@ -1071,8 +1071,8 @@
     #define configAPPLICATION_ALLOCATED_HEAP    0
 #endif
 
-#ifndef configENABLE_HEAP_PROTECTOR
-    #define configENABLE_HEAP_PROTECTOR    0
+#ifndef configENABLE_HEAP_PROTECTION
```

Comment:
I like the modifications, but the name "HEAP_PROTECTOR" was to be consistent with GCC and Clang's stack protection flag "-fstack-protector". Since the two are very functionally similar -- with the difference that one implements canaries to protect the stack, and the other canaries to protect heap block pointers -- I was thinking using a similar name would make it clear to users what the intended function is.

Reply:
Changed to protector.